sqlite_query.c

```diff
@@ -111,15 +131,27 @@ sqlite_convert_to_pg(Oid pgtyp, int pgtypmod, sqlite3_stmt * stmt, int stmt_coli
 				valstr = DatumGetCString(DirectFunctionCall1(float8out, Float8GetDatum((float8) value)));
 				break;
 			}
+		/* some popular datatypes for default algorythm branch
+		 * case BPCHAROID:
+		 * case VARCHAROID:
+		 * case TEXTOID:
+		 * case JSONOID:
+		 * case NAMEOID:
+		 * case TIMEOID:
+		 */
 		default:
-			valstr = (char *) sqlite3_column_text(stmt, stmt_colid);
+			{	/*
```

Comment:
Please enter the comment to the new line.

Reply:
Changed

Reply:
Thank you.